packages/runtime/src/runtime.ts

```diff
@@ -155,13 +148,13 @@ export class Runtime {
     for (const acc of accounts) {
       this.store.accounts.set(acc.address, acc);
 
-      acc.createdApps.forEach((params, id) => {
-        this.store.globalApps.set(id, params);
-      });
+      for (const appId of acc.createdApps.keys()) {
+        this.store.globalApps.set(appId, acc.address);
+      }
 
-      acc.createdAssets.forEach((params, assetIndex) => {
-        this.store.assetDefs.set(assetIndex, params);
-      });
+      for (const assetId of acc.createdAssets.keys()) {
+        this.store.assetDefs.set(assetId, acc.address);
+      }
```

Comment:
We have `initializeAccounts()` in runtime constructor. But currently we create an app/asset after runtime initialization. eg.
```
const runtime = new Runtime([john, bob]); // sets this.store.accounts
runtime.addApp({..}), runtime.updateApp({..})..etc afterwards. 
```
Since we don't have any created apps or assets during initialization of runtime, do lines 151-153, 155-157 ever get executed?

Reply:
yes, they do, when we call the constructor new Runtime([john, bob]). and if john and bob, have createdAssets or createdApps, we are mocking these.

Reply:
how can john or bob create new asset or app before initializing runtime?

Reply:
If you see we have mocks, which has hard coded createdAssets, this is used for that,
I haven't structured it like that, I think it was added by you. We can update in new PR

Reply:
I think it's good for user, user can send updated states if he wishes.
We can discuss it's pros and cons.

Reply:
I wasn't saying to remove it, was rather curious when this does gets executed.
For mocks the structure seems fine, as we set in globalassets and App directly there. From users perspective, I only see this code getting executed when the user explicitly modifies createdAssets or Map by using account.createdApps.set(..)..